accounting/rewind.go

```diff
@@ -99,7 +99,13 @@ func AccountAtRound(ctx context.Context, account models.Account, round uint64, d
 		MinRound: round + 1,
 		MaxRound: account.Round,
 	}
+	ctx, cf := context.WithCancel(ctx)
 	txns, r := db.Transactions(ctx, tf)
+	defer func() {
+		cf()
+		for range txns {
+		}
+	}()
```

Comment:
why not just defer cf()? what is line 106 for?

Reply:
I'm not 100% sure, but I think that `txns` is hung while trying to write the next result into it, so we need to discard the final entries in order for it to terminate.